components/server/src/prebuilds/bitbucket-app.ts

```diff
@@ -205,9 +211,11 @@ export class BitbucketApp {
                 if (teamMembers.some((t) => t.userId === webhookInstaller.id)) {
                     return { user: webhookInstaller, project };
                 }
+                const hostContext = this.hostCtxProvider.get(new URL(cloneURL).host);
+                const authProviderId = hostContext?.authProvider.authProviderId;
                 for (const teamMember of teamMembers) {
                     const user = await this.userService.findUserById(teamMember.userId, teamMember.userId);
-                    if (user && user.identities.some((i) => i.authProviderId === "Public-Bitbucket")) {
```

Comment:
oh, how did this work before?

Reply:
Good question, @svenefftinge!

See this is the second best strategy and is only evaluated if the webhook installer isn't under the org members anymore, cf. `return { user: webhookInstaller, project }` a couple of lines above this.

But still, this is a bug. 

I wanted to extract `findProjectAndOwner` and move to `ProjectService`, but that caused so many changes, I'd rather want to make a separate PR of it.